packages/amplication-server/src/core/serviceSettings/dto/ServerSettingsUpdateInput.ts

```diff
@@ -18,4 +18,7 @@ export class ServerSettingsUpdateInput {
     nullable: true,
   })
   serverPath?: string;
+
+  @Field(() => Boolean, { nullable: true })
```

Comment:
Showing the default value might be a good idea:

```suggestion
  @Field(() => Boolean, { nullable: true, defaultValue: true })
```